sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob/ContainerAsyncClientJavaDocCodeSnippets.java

```diff
@@ -81,4 +103,487 @@ public void generateSASCodeSnippets() {
+     */
+    public void getSnapshotAppendBlobAsyncClient() {
+        // BEGIN: com.azure.storage.blob.ContainerAsyncClient.getAppendBlobAsyncClient#String-String
+        AppendBlobAsyncClient appendBlobAsyncClient = client.getAppendBlobAsyncClient(blobName, snapshot);
+        // END: com.azure.storage.blob.ContainerAsyncClient.getAppendBlobAsyncClient#String-String
+    }
+
+    /**
+     * Code snippet for {@link ContainerAsyncClient#getBlockBlobAsyncClient(String)}
+     */
+    public void getBlockBlobAsyncClient() {
+        // BEGIN: com.azure.storage.blob.ContainerAsyncClient.getBlockBlobAsyncClient#String
+        BlockBlobAsyncClient blockBlobAsyncClient = client.getBlockBlobAsyncClient(blobName);
+        // END: com.azure.storage.blob.ContainerAsyncClient.getBlockBlobAsyncClient#String
+    }
+
+    /**
+     * Code snippet for {@link ContainerAsyncClient#getBlockBlobAsyncClient(String, String)}
+     */
+    public void getSnapshotBlockBlobAsyncClient() {
+        // BEGIN: com.azure.storage.blob.ContainerAsyncClient.getBlockBlobAsyncClient#String-String
+        BlockBlobAsyncClient blockBlobAsyncClient = client.getBlockBlobAsyncClient(blobName, snapshot);
+        // END: com.azure.storage.blob.ContainerAsyncClient.getBlockBlobAsyncClient#String-String
+    }
+
+    /**
+     * Code snippet for {@link ContainerAsyncClient#getPageBlobAsyncClient(String)}
+     */
+    public void getPageBlobAsyncClient() {
+        // BEGIN: com.azure.storage.blob.ContainerAsyncClient.getPageBlobAsyncClient#String
+        PageBlobAsyncClient pageBlobAsyncClient = client.getPageBlobAsyncClient(blobName);
+        // END: com.azure.storage.blob.ContainerAsyncClient.getPageBlobAsyncClient#String
+    }
+
+    /**
+     * Code snippet for {@link ContainerAsyncClient#getPageBlobAsyncClient(String, String)}
+     */
+    public void getSnapshotPageBlobAsyncClient() {
+        // BEGIN: com.azure.storage.blob.ContainerAsyncClient.getPageBlobAsyncClient#String-String
+        PageBlobAsyncClient pageBlobAsyncClient = client.getPageBlobAsyncClient(blobName, snapshot);
+        // END: com.azure.storage.blob.ContainerAsyncClient.getPageBlobAsyncClient#String-String
+    }
+
+    /**
+     * Code snippet for {@link ContainerAsyncClient#exists()}
+     */
+    public void exists() {
+        // BEGIN: com.azure.storage.blob.ContainerAsyncClient.exists
+        client.exists().subscribe(response -> System.out.printf("Exists? %b%n", response.booleanValue()));
+        // END: com.azure.storage.blob.ContainerAsyncClient.exists
+    }
+
+    /**
+     * Code snippet for {@link ContainerAsyncClient#existsWithResponse()}
+     */
+    public void existsWithResponse() {
+        // BEGIN: com.azure.storage.blob.ContainerAsyncClient.existsWithResponse
+        client.existsWithResponse().subscribe(response -> System.out.printf("Exists? %b%n", response.value().booleanValue()));
```

Comment:
Causing Javadoc generation issues since the line is over 120 characters. Also for all these existence calls can we just use `.value()` as Boolean toString will implicitly use the boolean value.

Reply:
updated